alphagov/paas-cf-plugin
# [#152237982] conduit: service tunnels for all

## What

A plugin for creating tunnels to service instances and making it easier to perform database maintaince tasks (such as using `psql` or `mysqldump`).

## How to review

* Read the [README](https://github.com/alphagov/paas-cf-plugin/blob/152237982-easier-db-interaction/README.md) for install instructions
* Create postgres and mysql service instances in your environment (if you have none)
* Try creating a basic tunnel `cf conduit your-instance` and check that the output instructions are clear enough
* Try using some postgres tools (ie `psql` or `pg_dump` examples in README)
* Try using some mysql tools (ie `mysql or `mysqldump` examples in README)
* OutOfScope/Untested but probably works... try connecting an app that wants VCAP_SERVICES running locally via: `cf conduit your-instance-1 your-instance-2 -- /path/to/your_awesome_app`
* General code reviewage

## Notes

Windows support is an unknown at this stage... but may JustWork

## Who can review

Not @chrisfarms nor @46bit
